cloudplatform/cloudplatform-connectivity-scp-cf/src/test/java/com/sap/cloud/sdk/cloudplatform/connectivity/ScpCfDestinationLoaderCacheTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2023 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+package com.sap.cloud.sdk.cloudplatform.connectivity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import java.time.Duration;
+
+import org.junit.After;
+import org.junit.Test;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.sap.cloud.sdk.cloudplatform.cache.CacheKey;
+import com.sap.cloud.sdk.cloudplatform.resilience.CacheExpirationStrategy;
+
+import io.vavr.control.Option;
+
+public class ScpCfDestinationLoaderCacheTest
```

Comment:
```suggestion
public class DestinationServiceCacheTest
```
According to the renaming we did yesterday ;)